app/api/main.py

```diff
@@ -1,10 +1,16 @@
-from fastapi import APIRouter
+from fastapi import APIRouter, Depends
 
-from app.api.routes import items, items_service, users
+from app.api.routes import google, items, items_service, users
+from app.utilities.dependencies import get_token_header
 
-api_router = APIRouter()
-api_router.include_router(users.router, prefix="/users", tags=["users"])
-api_router.include_router(items.router, prefix="/items", tags=["items"])
-api_router.include_router(
+api_router_open = APIRouter()
+api_router_open.include_router(google.router, prefix="/google", tags=["google"])
+
+api_router_token = APIRouter(
+    dependencies=[Depends(get_token_header)],
+)
```

Comment:
Les cambiaría el nombre. Si bien se puede intuir esta raro el `open` y el `token` también, por que lo que se le pasa es una key. Seguramente cuando hagamos el agregado con JWT (token), se nos van a mezclar

```suggestion
router_without_api_key = APIRouter()
router_without_api_key.include_router(google.router, prefix="/google", tags=["google"])

router_with_api_key = APIRouter(
    dependencies=[Depends(get_token_header)],
)
```

Reply:
Procedo a realizar las moficaciones solcitadas